CHANGELOG.md

```diff
@@ -1,11 +1,33 @@
 # Changelog - Fyne.io fyne-cross
 
-## Unreleased
+## [1.1.0] Unreleased
+
+### Added
+
+- Add FreeBSD on arm64 target #29
+- Add the `darwin-image` command to build the darwin docker image
+- Add a dedicated docker image for windows
```

Comment:
Don't we have dedicated images for basically everything now?
Also, it should be "Windows".